python/tests/engine/test_python.py

```diff
@@ -1752,6 +1753,24 @@ def test_convert_to_unix_timestamp_int(self):
         # Assert
         assert result == 1483225200000
 
+    def test_convert_to_unix_timestamp_datetime(self):
+        # Act
+        result = util.convert_event_time_to_timestamp(
+            event_time=datetime(2022, 9, 18), time_zone=timezone.utc
```

Comment:
A test for pd.Timestamp is still missing. In best case the tests would also cover a datetime/timestamps with tzinfo set. e.g. `datetime(2022, 7, 3, 0, 0, 0).replace(tzinfo=timezone(timedelta(hours=3)))` and `pd.Timestamp("2022-07-03T00", tz="US/Pacific")`